datasets/qasper/README.md

```diff
@@ -0,0 +1,204 @@
+    - [Other Known Limitations](#other-known-limitations)
+  - [Additional Information](#additional-information)
+    - [Dataset Curators](#dataset-curators)
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+    - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** [https://allenai.org/data/qasper](https://allenai.org/data/qasper)
+- **Demo:** [https://qasper-demo.apps.allenai.org/](https://qasper-demo.apps.allenai.org/)
+- **Paper:** [https://arxiv.org/abs/2105.03011](https://arxiv.org/abs/2105.03011)
+- **Blogpost:** [https://medium.com/ai2-blog/question-answering-on-scientific-research-papers-f6d6da9fd55c](https://medium.com/ai2-blog/question-answering-on-scientific-research-papers-f6d6da9fd55c)
+- **Leaderboards:** [https://paperswithcode.com/dataset/qasper](https://paperswithcode.com/dataset/qasper)
+
+### Dataset Summary
+
+QASPER is a dataset for question answering on scientific research papers. It consists of 5,049 questions over 1,585 Natural Language Processing papers. Each question is written by an NLP practitioner who read only the title and abstract of the corresponding paper, and the question seeks information present in the full text. The questions are then answered by a separate set of NLP practitioners who also provide supporting evidence to answers.
+
+### Supported Tasks and Leaderboards
+
+- `question-answering`: The dataset can be used to train a model for Question Answering. Success on this task is typically measured by achieving a *high* [F1 score](https://huggingface.co/metrics/f1). The [official baseline model](https://github.com/allenai/qasper-led-baseline) currently achieves 33.63 Token F1 score & uses [Longformer](https://huggingface.co/transformers/model_doc/longformer.html). This task has an active leaderboard which can be found [here](https://paperswithcode.com/sota/question-answering-on-qasper) 
+
+- `evidence-selection`: The dataset can be used to train a model for Evidence Selection. Success on this task is typically measured by achieving a *high* [F1 score](https://huggingface.co/metrics/f1). The [official baseline model](https://github.com/allenai/qasper-led-baseline) currently achieves 39.37 F1 score & uses [Longformer](https://huggingface.co/transformers/model_doc/longformer.html). This task has an active leaderboard which can be found [here](https://paperswithcode.com/sota/evidence-selection-on-qasper) 
+
+
+### Languages
+
+English, as it is used in research papers.
+
+## Dataset Structure
+
+### Data Instances
+
+A typical instance in the dataset:
+
+```
+{
+  'id': "Paper ID (string)",
+  'title': "Paper Title",
+  'abstract': "paper abstract ...",
+  'full_text': {
+      'paragraphs':[["section1_paragraph1_text","section1_paragraph2_text",...],["section2_paragraph1_text","section2_paragraph2_text",...]],
+      'section_name':["section1_title","section2_title"],...},
+  'qas': [explained below],
+}
+```
```

Comment:
Could we have an actual example here instead ?

Since it can be long, feel free to crop some parts

Reply:
an actual example was too verbose and hard to understand no matter how I cropped/simplified it. imo an abstract version is easier to understand at first look.
I added some more explanations to the `qas` key and how it lists questions and its answers. Since The pyarrow format can feel confusing at first.
Let me know if this is okay